include/aos/common/tools/time.hpp

```diff
@@ -239,10 +239,10 @@ class Time {
      */
     friend Log& operator<<(Log& log, const Time& obj)
     {
-        static constexpr auto cTimeStrLen = 32;
+        static constexpr auto cUTCTimeStrLen = 32;
```

Comment:
Why it is renamed?

Reply:
This constant shadows cTimeStrLen declared in the aos/comon/tools/time.hpp

Reply:
Then cTimeStrLen should be used instead, no need to add new one. They look the same.

Reply:
cTimeStrLen == 68
author of this code specified that string size will be 32.

So I see 2 options:
1. simply use cTimeStrLen global constant (68 bytes)
2. add a new constant (like AOS_CONFIG_TIME_UTS_STR_LEN 32) and use it

Reply:
reused global constant.